starboard/nplb/player_test_util.cc

```diff
@@ -147,6 +147,44 @@ std::vector<SbPlayerTestConfig> GetSupportedSbPlayerTestConfigs(
   return test_configs;
 }
 
+std::vector<SbPlayerTestConfig> GetVerticalVideoTestConfigs() {
+  const char* kEmptyName = NULL;
+
+  // TODO: Add these files.
+  const char* kFilenames[] = {"1080_1920_avc.dmp", "1440_2560_av1.dmp",
+                              "3840_2160_av1.dmp"};
```

Comment:
Is it intended to add "3840_2160_av1.dmp" here?

And one question about why we choose 1080p avc, 2k and 4k av1, with no vp9?

Reply:
This will change to: 1080p avc, 2k vp9, 4k vp9, and 8k av1. The 4K one should be "2160_3840.dmp"

Reply:
For now I've added 1080p avc, 4k vp9, and 8k av1. I'll generate and add 2k vp9 sometime later.